src/main/webapp/dev/js/common/const.js

```diff
@@ -2,24 +2,54 @@
  * Contains constants to be used across the application.
  */
 
-// Status message type
-const StatusType = {
+// Frontend only constants
+const Const = {
+
+    ModalDialog: {
+
+        UNREGISTERED_STUDENT: {
+            header: 'Register for TEAMMATES',
+            text: 'You have to register using a google account in order to access this page. '
+                  + 'Would you like to proceed and register?',
+        },
+
+    },
+
+    StatusMessages: {
+
+        INSTRUCTOR_DETAILS_LENGTH_INVALID: 'Instructor Details must have 3 columns',
+
+    },
+
+};
+
+// Shared constants between frontend and backend
+
+// Subset of Bootstrap contextual colors for use in status messages and components of modals
+const BootstrapContextualColors = {
+    // Mirrored colors from StatusMessageColor
     SUCCESS: 'success',
     INFO: 'info',
     WARNING: 'warning',
     DANGER: 'danger',
+    // Additional contextual colors that can be used in the components of modals
     PRIMARY: 'primary',
     isValidType(type) {
-        return type === StatusType.SUCCESS || type === StatusType.INFO || type === StatusType.PRIMARY
-               || type === StatusType.WARNING || type === StatusType.DANGER;
+        return type === BootstrapContextualColors.SUCCESS || type === BootstrapContextualColors.INFO ||
+        type === BootstrapContextualColors.PRIMARY || type === BootstrapContextualColors.WARNING ||
+        type === BootstrapContextualColors.DANGER;
     },
 };
-StatusType.DEFAULT = StatusType.INFO;
+BootstrapContextualColors.DEFAULT = BootstrapContextualColors.INFO;
 
+// Mirrored subset of Const#ParamNames
 const ParamsNames = {
```

Comment:
Fix the order?

Reply:
The order has to be the same as that in Const#ParamsNames right ?
But there is no FEEDBACK_QUESTION_CONSTSUMOPTIONTABLE, FEEDBACK_QUESTION_RANKOPTIONTABLE in Const#ParamsNames, though it is in const.js
Except that, I've tried to order everything. Please check. :)

Reply:
Not your fault, but please inline the two constants so we can delete them.

Thanks a lot for your effort :).

Reply:
Can you please elaborate on what you mean by "inline the two constants" ?
Thanks a lot for your guidance.
Couldn't have done it till here without you. :)

Reply:
For example,

```Javascript
function hideRankOptionTable(questionNum) {
    $(`#${ParamsNames.FEEDBACK_QUESTION_RANKOPTIONTABLE}-${questionNum}`).hide();
}
```

to
```
function hideRankOptionTable(questionNum) {
    $(`#rankOptionTable-${questionNum}`).hide();
}
```

Reply:
@LiHaoTan I've tried to make them inline. Please check.